tests/test-worker-url.js

```diff
@@ -0,0 +1,18 @@
+import assert from 'tjs:assert';
+
+const data = JSON.stringify({ foo: 42, bar: 'baz!' });
+const worker = `if (self.addSignalListener || self.removeSignalListener) {throw new Error('There are signals in a Worker!');} self.postMessage(${data});`
+const blob = new Blob([worker], { type: 'text/javascript' });
+const url = URL.createObjectURL(blob);
+
+const w = new Worker(url);
+URL.revokeObjectURL(url);
```

Comment:
There is a problem that I cannot solve through the api or any tricks. It can happen that the `URL.revokeObjectURL`               is processed before the creation of the worker. Consequently, the blob will not be found when it is required during the `TJS_EvalModule`.

I'm a bit stuck at that point.

Reply:
Indeed, I could instead put the work in the Worker instead of the EvalModule. That was to keep intact the Worker polyfill.

Reply:
I reckon it's a safer option!